packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.Props.ts

```diff
@@ -74,6 +75,11 @@ export interface ISwatchColorPickerProps {
    * Should focus cycle to the beginning of once the user navigates past the end (and visa vsersa). Defaults to true
    */
   shouldFocusCircularNavigate?: boolean;
+
+  /**
+ * Theme provided by HOC.
+ */
+  theme?: ITheme;
```

Comment:
I don't think this is necessary. The component itself shouldn't need to know if the colors come from a theme.

Reply:
I talked to David and he agrees that I need to use customizable for this, in order to listen to theme changes.